src/java/htsjdk/variant/variantcontext/GenotypesContext.java

```diff
@@ -39,7 +40,9 @@
 /**
  * Represents an ordered collection of Genotype objects
  */
-public class GenotypesContext implements List<Genotype> {
+public class GenotypesContext implements List<Genotype>, Serializable {
```

Comment:
this class has a subclass LazyGenotypesContext that has a field `Object unparsedGenotypeData;` - this is not serializable

It also has a field of type LazyParser that has 2 implementations:
- one is a member inner class of AbstractVCFCodec (and serializing it would pull in all of AbstractVCFCodec and its superclasses)
- the other is a BCF2LazyGenotypesDecoder which refers to a lot of stuff which would need to be serializable